### PR TITLE
Attribute Listener interface changed

### PR DESCRIPTION
**Pull request by mrpollo**
- vehicle is also an attribute to expect on the listener interface
- test environment with solo sitl
- tests the attribute listener interface to avoid future conflicts

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+dronekit==2.1.0
+dronekit-sitl==3.0.1
+nose==1.3.7
+psutil==3.4.2
+pymavlink==1.1.70
+requests==2.9.1
```
Comment (tcr3dr) [inline requirements.txt]: Do you need requests?

**Comment by tcr3dr** [comment]
This lgtm after you pull in #2 as well :)

File: requirements.txt (hunk 1 above)
Comment (mrpollo) [inline requirements.txt]: i didn't explicitly install requests might be a dep from something else, ill remove it

**Comment by mrpollo** [comment]
just rebased #2 into this

**Comment by wsilva32** [comment]
Also #3.

**Comment by tcr3dr** [comment]
lgtm

**Pull request merged**